variables.tf

```diff
@@ -308,3 +308,225 @@ variable "check_ec2_imdsv2" {
   type        = bool
   default     = false
 }
+
+variable "check_access_keys_rotated" {
+  description = "Enable access-keys-rotated rule"
+  type        = bool
+  default     = true
+}
+
+variable "access_key_max_age" {
+  description = "Maximum number of days without rotation."
+  type        = number
+  default     = 90
+}
+
+variable "check_cmk_backing_key_rotated" {
+  description = "Enable cmk_backing_key_rotation_enabled rule"
+  type        = bool
+  default     = true
+}
+
+variable "check_backup_plan_min_frequency_and_min_retention" {
+  description = "Enable backup_plan_min_frequency_and_min_retention_check rule"
+  type        = bool
+  default     = false
+}
+
+variable "backup_frequency" {
+  description = "Numerical value for required backup frequency. Maximum of 24 for hours, 31 for days."
+  type        = number
+  default     = 1
```

Comment:
Is this 1 hour? 1 day?

Reply:
To keep with consistency, all descriptions for variables that are parameters of a rule are pulled directly from AWS dev docs. in this instance, there is another parameter that denoted this value with a default of "days" but can be changed to hours.

`variable "backup_frequency_unit_of_time" {
  description = "Unit of time for required backup frequency. Accepted values: 'hours', 'days'."
  type        = string
  default     = "days"
}`

Reply:
We should document the units in the description, even if AWS doesn't.

Reply:
additional description added

Reply:
I wonder if it's worth mentioning that `days` is set via the variable `backup_frequency_unit_of_time` below. There is _a lot_ in this and it's possible a person may only skim this and be unaware of the value below.

Reply:
This is the new description I added last push. 
`description = "Numerical value for required backup frequency. Default value of 'backup_frequency_unit_of_time' is days. Maximum of 24 for hours, 31 for days."`